resources/constraints/constraints_py3.10.txt

```diff
@@ -169,23 +180,32 @@ mpmath==1.3.0
     # via sympy
 mypy-extensions==1.0.0
     # via psygnal
+napari==0.4.15
+    # via napari-svg
```

Comment:
it looks like `napari-svg` but I do not understand why it resolve to 0.4.15 https://github.com/napari/napari-svg/pull/33

Reply:
Ok. The upper bounds is present in 0.4.17 and 0.4.16, so it resolves to 0.4.15